app/assets/stylesheets/homepage.css.scss

```diff
@@ -68,8 +68,8 @@
     font-family: 'Sans-Serif';
     background: green;
     box-shadow: white 1px 1px 1px;
-    top: 10px;
-    right: 7px;
+    top: 13px;
+    right: 0px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>`0px` should be written without units as `0`